assembly/assembly-main/src/assembly/tomcat/conf/server.xml

```diff
@@ -139,6 +139,10 @@
       <Host name="localhost"  appBase="webapps"
             unpackWARs="true" autoDeploy="false">
 
+
+        <!-- Provide Che Agent binaries (like workspace agent, terminal) -->
+        <Context docBase="${che.home}/lib" path="/agent-binaries" />
```

Comment:
separate folder by duplicating the existing binaries ? because agents are inside CHE_HOME/lib for now